src/logic/unique.lean

```diff
@@ -151,3 +151,15 @@ protected def injective.unique [inhabited α] [subsingleton β] (hf : injective
 @unique.mk' _ _ hf.subsingleton
 
 end function
+
+section has_elem
+
+@[simp] lemma unique_has_one {α : Type*} [unique α] [has_one α] :
+  default α = 1 :=
+unique.default_eq 1
+
+@[simp] lemma unique_has_zero {α : Type*} [unique α] [has_zero α] :
+  default α = 0 :=
+unique.default_eq 0
```

Comment:
`to_additive`?

Reply:
It's too early in the import hierarchy for `to_additive`.

Reply:
Then you should add the attribute in the file that provides `to_additive`

Reply:
Actually - why not just move this to `algebra.group.basic`? No one is going to want these lemmas without some knowledge about 0 and 1 anyway, realistically.

Also, which simp lemma is picked when working with `ring α`?